src/js/themes/hpe.js

```diff
@@ -1984,7 +2107,16 @@ const buildTheme = (tokens, flags) => {
         ],
         columns: [['small', 'flex'], 'auto'],
         rows: ['auto', 'auto', 'auto'],
-        gap: { row: 'xsmall', column: 'medium' },
+        gap: { row: '3xsmall', column: 'medium' },
+      },
+      small: {
+        gap: { row: '3xsmall', column: 'large' },
+      },
+      medium: {
+        gap: { row: '3xsmall', column: 'medium' },
+      },
+      large: {
+        gap: { row: '3xsmall', column: 'large' },
```

Comment:
Should column be 'xlarge' here? Looks like it is 'large' in base.js
```suggestion
        gap: { row: '3xsmall', column: 'xlarge' },
```